conkit/command_line/conkit_validate.py

```diff
@@ -0,0 +1,253 @@
+
+
+def prepare_output_directory(outdir, overwrite=False):
+    """Prepare the output directory for conkit-validate.
+
+    Parameters
+    ----------
+    outdir : str
+       Path to the output directory
+    overwrite : bool
+       Whether the output directory should be overwritten or not [default: False]
+
+    Raises
+    ------
+    :exc:`ValueError`
+        The output directory already exists and overwrite is False
+    """
+
+    if os.path.isdir(outdir):
+        if not overwrite:
+            raise ValueError('Output directory already exists')
+        else:
+            shutil.rmtree(outdir)
+    os.mkdir(outdir)
+
+
+def parse_map_align_stdout(stdout):
+    """Parse the stdout of map_align and extract the alignment of residues.
+
+    Parameters
+    ----------
+    stdout : str
+       Starndard output created with map_align
+
+    Returns
+    ------
+    alignment: dict
+        A dictionary where the aligned residue numbers of map_b are the keys and the residue numbers of map_a the values
+    """
+
+    alignment = {}
+    for line in stdout.split('\n'):
+        if line and line.split()[0] == "MAX":
+            line = line.rstrip().lstrip().split()
+            for residue in line[8:]:
+                resnum = residue.split(":")
+                alignment[int(resnum[1])] = int(resnum[0])
+
+    return alignment
+
+
+def main():
+    """The main routine for conkit-validate functionality"""
+    parser = create_argument_parser()
+    args = parser.parse_args()
+
+    global logger
+    logger = conkit.command_line.setup_logging(level="info")
+
+    logger.info("Output directory:                           %s", args.outdir)
+
+    prepare_output_directory(args.outdir, args.overwrite)
+
+    sequence = conkit.io.read(args.seqfile, args.seqformat).top
+    prediction = conkit.io.read(args.distfile, args.distformat).top
+    model = conkit.io.read(args.pdbfile, args.pdbformat).top
+    logger.info("Input PDB model:                            %s", args.pdbfile)
+    logger.info("Input distance prediction:                  %s", args.distfile)
+    logger.info("Length of the sequence:                     %d", len(sequence))
+
+    fig_fname = os.path.join(args.outdir, 'conkit.png')
+    figure = conkit.plot.ModelValidationFigure(model, prediction, sequence, use_weights=True)
+    figure.savefig(fig_fname, overwrite=args.overwrite)
+    logger.info("Validation plot written to %s", fig_fname)
+
+    if args.skip_alignment:
+        logger.info("Skipping contact map alignment, no changes suggested.")
+
+    elif any(figure.outliers):
+        table = PrettyTable()
+        table.field_names = ["Outlier no.", "Residue no.", "wRMSD", "FN Count"]
+        for idx, outlier in enumerate(figure.outliers, 1):
+            table.add_row([idx, str(outlier), '{0:.2f}'.format(figure.rmsd_profile[outlier]),
+                           '{0:.2f}'.format(figure.fn_profile[outlier])])
+        logger.info("\nList of detected outliers:")
```

Comment:
Is this conkit windows compatible? Might be worth swapping `\n` with `os.linesep`

Reply:
Yes that's a good point, I didn't realise that conkit was also available for windows... Not sure if `map_align` is available for windows, but anyway I'll make the change